src/events.js

```diff
@@ -1,7 +1,11 @@
-/*
- * A list of DOM events can be found on MDN:
+/**
+ * List of DOM events to be tracked.
+ * <br/>
+ * An exhaustive list of DOM events can be found on MDN:
  *   https://developer.mozilla.org/en-US/docs/Web/Events
```

Comment:
I'd suggest a br before/after the link, looks better in built JSDoc.

Reply:
Done, plus put the link a proper `<a>...</a>`.